common/workunit/workunit.hpp

```diff
@@ -65,6 +65,8 @@ typedef unsigned short UChar;
 #define QUERRREG_UNSUSPEND          QUERYREGISTRY_ERROR_START+9
 #define QUERRREG_COMMENT            QUERYREGISTRY_ERROR_START+10
 
+#define ECLAGENTLOGSEARCHSTR        "eclagent."
```

Comment:
only internal, only used by implementation in workunit.cpp?

Reply:
For now, yes.

Reply:
ok, I think should remain an implementation detail only, and so should really be moved out of header into implementation.

Reply:
I will move this line to workunit.cpp.